python/lsst/meas/base/plugins.py

```diff
@@ -371,6 +372,38 @@ def fail(self, measRecord, error=None):
         pass
 
 
+class SingleFrameFootprintAreaConfig(SingleFramePluginConfig):
+    pass
+
+
+@register("base_FootprintArea")
+class SingleFrameFootprintAreaPlugin(SingleFramePlugin):
+    """A measurement plugin that simply records the area of the source's
+    detection Footprint.
+    """
+
+    def __init__(self, config, name, schema, metadata):
+        SingleFramePlugin.__init__(self, config, name, schema, metadata)
+        self.key = schema.addField(
+            schema.join(name, "value"),
+            type=numpy.int32,
+            doc="Number of pixels in the source's detection footprint.",
+            units="pixel"
+        )
+
+    @classmethod
+    def getExecutionOrder(cls):
+        return cls.CENTROID_ORDER  # probably doesn't need to go early, but can
```

Comment:
If comments are statements are not allowed it's news to me (but it may have been a change brought on by adopting more of PEP8, which I haven't followed closely). In any case, this is now moot since I've rewritten this as a `CatalogCalculation` and I'll just use the default execution order for those.